src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/library_db";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "13579Wryip+";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```

Comment:
Consider moving the database credentials to a configuration file for better security and configurability. 🛡️